src/libs/actions/Policy.js

```diff
@@ -22,8 +22,17 @@ Onyx.connect({
         if (!key) {
             return;
         }
-
         if (val === null || val === undefined) {
+            const policyID = key.replace(ONYXKEYS.COLLECTION.POLICY, '');
```

Comment:
Would you mind adding a code comment to explain why this logic is here?

Reply:
I'm not sure what to write, it's pretty clear once you know the value of `ONYXKEYS.COLLECTION.POLICY`.
The `key` is something like: `policy_1234`
and the collection name is `policy_`

Removing the `policy_` part from the `key` gets us the `policyID` (1234)

Reply:
Oh, I meant a little bit more generally speaking. Think of it like answering the question: "why are we doing something with reports when a policy is deleted?"

Reply:
I think that's how we do it across the app
https://github.com/Expensify/App/blob/a109d855383ee1d6ac8da078fc9a7b81968ed868/src/libs/actions/User.js#L266

Reply:
Updated